products/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+from django.contrib.auth import get_user_model
+from Bogdans_shop.models import CustomModel
+
+AuthUserModel = get_user_model()
+
+# Create your models here.
+
+
+class Products(CustomModel):
+    user = models.ForeignKey(AuthUserModel, on_delete=models.CASCADE)
+    name = models.CharField(max_length=255, null=False)
+    price = models.DecimalField(max_digits=8, decimal_places=2, default=0.00)
+    auction_price = models.DecimalField(max_digits=8, decimal_places=2, default=0.00)
+    validation = models.BooleanField(default=False)
```

Comment:
Is this field about the product been approved or not by an admin? If so, I would use `is_approved` instead of `validation`.